Formula/u/unarj.rb

```diff
@@ -4,6 +4,11 @@ class Unarj < Formula
   url "https://src.fedoraproject.org/repo/pkgs/unarj/unarj-2.65.tar.gz/c6fe45db1741f97155c7def322aa74aa/unarj-2.65.tar.gz"
   sha256 "d7dcc325160af6eb2956f5cb53a002edb2d833e4bb17846669f92ba0ce3f0264"
 
+  livecheck do
+    url "https://src.fedoraproject.org/repo/pkgs/unarj/"
```

Comment:
I think Fedora dropped `unarj` so I don't expect this to ever update (if the upstream is even active).

Still need to parse through terms of license, but may be too prohibitive for usage in Homebrew/core.